gpt_bpe.go

```diff
@@ -64,6 +64,7 @@ type GPTEncoder struct {
 	encloseEos            bool
 	prefixSpace           bool
 	lowerCase             bool
+	spaceMerge            bool
```

Comment:
Rather than hardcode it in a configuration file, we should consider behavioral detection. What's unique about the glm vocabulary that we need to do this behavior?